tests/test_assignment_materialize_520.py

```
@@ -0,0 +1,964 @@
+    payload = {
+        "kind": "assignment",
+        "title": "核钱粮",
+        "target_id": "he-qianliang",
+        "assignee": other.name,  # 分类器误填他人
+        "name": other.name,
+        "commitment_kind": "无",
+    }
+    candidates = candidates_from_classifier_payload(payload, soft=False)
+    ctx = _ctx(
+        db, actor.name, candidates, state.turn,
+        message=f"这核钱粮的事你办。",
```

Ruff (0.16.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)